org-code-javabuilder/protocol/src/main/java/org/code/protocol/CachedResources.java

```diff
@@ -0,0 +1,49 @@
+package org.code.protocol;
+
+import java.awt.*;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+// Resources we want to use across multiple invocations of Javabuilder.
+public class CachedResources {
+  private Map<String, Font> fontMap;
+
+  private static final String FONT_FOLDER_NAME = "fonts";
+  private static final String[] FONT_FILE_NAMES =
```

Comment:
These constants and `getFontMap()` feel like they might still belong in `FontHelper` (or something else new with font in the name), then we keep this object as part of this new `Cached Resources` class?

Reply:
see my comment on Jessie's comment, we have a limitation with not being able to reference media from protocol.